github_hooks/Makefile

```diff
@@ -88,7 +88,7 @@ check.ruby.code: check.prepare
 	$(MAKE) check.code LANGUAGE=ruby
 
 check.ruby.deps: check.prepare
-	$(MAKE) check.deps LANGUAGE=ruby
+	$(MAKE) check.deps LANGUAGE=ruby CHECK_DEPS_OPTS="-w rt-watchman,app"
```

Comment:
Instead of ignoring it, update the gem to the version with the License file. If you're having trouble doing that, please feel free to ping me on Slack, and I will help you.